### ISSUE TITLE
mat-menu trigger is setting aria-expanded always true after first menuOpen 

### ISSUE THREAD

**Issue by sarashaabanmodeso** [angular/components#26355]
### Is this a regression?

- [ ] Yes, this behavior used to work in the previous version

### Description

mat-menu trigger is setting aria-expanded always true after the first menuOpen,=

### Reproduction

Steps to reproduce:
1. 
        {{t.menuOpen}}
 <button mat-icon-button [matMenuTriggerFor]="menu" aria-label="Example icon-button with a menu">
  <mat-icon>more_vert</mat-icon>
</button>
<mat-menu #menu="matMenu">
  <button mat-menu-item>
    <mat-icon>dialpad</mat-icon>
    <span>Redial</span>
  </button>
  <button mat-menu-item disabled>
    <mat-icon>voicemail</mat-icon>
    <span>Check voice mail</span>
  </button>
  <button mat-menu-item>
    <mat-icon>notifications_off</mat-icon>
    <span>Disable alerts</span>
  </button>
</mat-menu>
3. check {{t.menuOpen}} result after close menu

### Expected Behavior

after closing menu aria-expanded= true should be null on mat-menu trigger

### Actual Behavior

mat-menu trigger  aria-expanded  attribute always true until next menu open 

### Environment

- Angular:14.0.7
- CDK/Material: ^14.2.1
- Browser(s): chrome 
- Operating System (e.g. Windows, macOS, Ubuntu): windows

**Comment by crisbeto** [comment]
Duplicate of https://github.com/angular/components/issues/26262.
